packages/odyssey-storybook/src/components/TextInput/TextInput.stories.tsx

```diff
@@ -50,8 +50,20 @@ export default {
 const Template: Story<TextInputProps> = (props) => <TextInput {...props} />;
 
 export const Text = Template.bind({});
-Text.args = {
-  defaultValue: "Jupiter",
+Text.args = {};
+
+export const Optional = Template.bind({});
+Optional.args = {
+  required: false,
+  optionalLabel: "Optional",
+};
+
+export const KitchenSink = Template.bind({});
+KitchenSink.args = {
```

Comment:
consider a story title that isn't as colloquial, something with meaning to a broader audience. Perhaps Survey or your use of Anatomy in the written docs above?